cmd/minishift/cmd/start_test.go

```diff
@@ -169,12 +173,13 @@ func TestStartClusterUpWithOverrideHostDataDirFlag(t *testing.T) {
 	defer viper.Reset()
 
 	viper.Set("host-data-dir", "/var/tmp/foo")
-	clusterUp(&testMachineConfig)
+	clusterUp(&testMachineConfig, "192.168.99.42")
```

Comment:
There are similar use of the IP string and not the `testIp` variable in below test cases also.